### ISSUE TITLE
Multiplayer?

### ISSUE THREAD

**Issue by VinnyVicious** [kripken/BananaBread#62]
I was checking the following BananaBread roadmap: https://wiki.mozilla.org/HTML5_Games/BananaBread#Milestone_Two

And it says that multiplayer with WebRTC is complete. Unfortunately, i wasn't able to find any docs or examples of how to setup a BananaBread server and have clients connecting to it. Does anyone know where i can find them?

**Comment by Croydon** [comment]
See also #57 & #40.

And yes I agree, if there aren't enough information available it's hard to find somebody who is setting up a server.

**Comment by kriekprinsloo** [comment]
How do you set up a server?

**Comment by VinnyVicious** [comment]
From what i could see, there's no dedicated server support. Only browser-based.

**Comment by TheCodeCrafter** [comment]
@kripken Question: Is the engine in any way in a current state to have multiplayer compatibility? We have the websock.js but can we use it?

**Comment by TheCodeCrafter** [comment]
I believe Websockets are much more efficient, anyways... So, what do you propose for implementation?

**Comment by Calinou** [comment]
> I believe Websockets are much more efficient, anyways...

Actually, WebRTC is more suited for real-time multiplayer games, since it can use unreliable communication (like UDP) whereas WebSockets are limited to reliable communication (like TCP), which suffers from higher overall latency.

**Comment by TheCodeCrafter** [comment]
That's true. I've just got more experience with Websockets so I guess I feel more inclined to use them... LOL
